src/nvidia-cuda/install.sh

```diff
@@ -33,6 +33,15 @@ check_packages() {
     fi
 }
```

Comment:
Can you bump the patch version of the Feature?

Reply:
done